nuxeo-management-rest-api/src/main/java/org/nuxeo/rest/management/ManagementRoot.java

```diff
@@ -32,8 +38,22 @@
 @WebObject(type = "management")
 public class ManagementRoot extends ModuleRoot {
 
+    public static final String MANAGEMENT_API_USER_PROPERTY = "org.nuxeo.rest.management.user";
+
     @Path("{path}")
     public Object route(@PathParam("path") String path) {
+        // check if the user can access the Management API
+        verifyUser();
+
         return newObject(path);
     }
+
+    protected void verifyUser() {
+        WebContext context = getContext();
+        NuxeoPrincipal principal = context.getPrincipal();
```

Comment:
`context` is only used once so you could just write `principal = getContext().getPrincipal()`